build-tools/packages/build-cli/src/lib/release.ts

```diff
@@ -0,0 +1,127 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import {
+    ReleaseVersion,
+    VersionBumpType,
+    VersionScheme,
+    detectVersionScheme,
+    getVersionRange,
+} from "@fluid-tools/version-tools";
+
+import { ReleaseGroup } from "../releaseGroups";
+
+export interface PackageVersionList {
+    [packageName: string]: string;
+}
+
+export interface ReleaseReport {
+    [packageName: string]: ReleaseDetails;
+}
+
+export interface ReleaseDetails {
+    version: ReleaseVersion;
+    previousVersion?: ReleaseVersion;
+    versionScheme: VersionScheme;
+    date?: Date;
+    releaseType: VersionBumpType;
+    isNewRelease: boolean;
+    releaseGroup?: ReleaseGroup;
+    ranges: ReleaseRanges;
+}
+
+export interface ReleaseRanges {
+    minor: string;
+    patch: string;
+    caret: string;
+    tilde: string;
+}
+
+export const getRanges = (version: ReleaseVersion, scheme?: VersionScheme): ReleaseRanges => {
+    // eslint-disable-next-line no-param-reassign
+    scheme = scheme ?? detectVersionScheme(version);
+    return scheme === "internal"
```

Comment:
I don't think it's worth violating our linter enforced policy here. Either declare another variable or do something like
```suggestion
    return (scheme ?? detectVersionScheme(version)) === "internal"
```

Reply:
Fixed in latest iteration.